### PR TITLE
226216-case: Unsubscribe from scheduled emails

### PR DESCRIPTION
**Pull request by pnaw94**

### CODE REVIEW

File: corehq/apps/reports/urls.py [hunk 1]
```diff
@@ -128,6 +128,8 @@
         ScheduledReportsView.as_view(), name=ScheduledReportsView.urlname),
     url(r'^scheduled_report/(?P<scheduled_report_id>[\w-]+)/delete$',
         'delete_scheduled_report', name='delete_scheduled_report'),
+    url(r'^scheduled_report/(?P<scheduled_report_id>[\w-]+)/unsubscribe', ReportNotificationUnsubscribeView.as_view(),
```
Comment (dimagimon) [inline corehq/apps/reports/urls.py]: E501 line too long (118 > 115 characters)

File: corehq/apps/reports/views.py [hunk 2]
```diff
@@ -3,9 +3,12 @@
 import itertools
 import json
 
+from django.views.generic.base import TemplateView
+
 from corehq.apps.app_manager.suite_xml.sections.entries import EntriesHelper
-from corehq.apps.domain.views import BaseDomainView
+from corehq.apps.domain.views import BaseDomainView, LoginAndDomainMixin
 from corehq.apps.hqwebapp.view_permissions import user_can_view_reports
+from corehq.apps.hqwebapp.views import redirect_to_default
```
Comment (dimagimon) [inline corehq/apps/reports/views.py]: F401 'redirect_to_default' imported but unused

**Comment by pnaw94** [comment]
@snopoke The functionality should be ok now. Let me know if I should make some more fancy layout for the view.

File: corehq/apps/reports/templates/reports/notification_unsubscribe.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+{% load i18n %}
+
+{% block title %}
+    {% trans 'Unsubscribe from emails' %}
+{% endblock %}
+
+{% block content %}
+    <form method="POST" style="margin-left: 20px">
+        {% csrf_token %}
+        <h2>{% blocktrans %}Are you sure you want to opt out of report's notifications?{% endblocktrans %}</h2>
```
Comment (snopoke) [inline corehq/apps/reports/templates/reports/notification_unsubscribe.html]: Is it possible to include the name of the report?

**Comment by snopoke** [comment]
Nice - one small comment

**Comment by pnaw94** [comment]
@snopoke updated

**Pull request merged**